### ISSUE TITLE
No 'annotationEditorEvent' for 'size-changed' and 'removed' with custom stamp

### ISSUE THREAD

**Issue by AdrianM** [stephanrauh/ngx-extended-pdf-viewer#2515]
**bug description**
When i dynamically add stamps (e.g. with `this.pdfServiceInstance.addImageToAnnotationLayer(annotation);` or `pdfServiceInstance.addEditorAnnotation(stampEditorAnnotation);`), i receive correctly the "annotationEditorEvent" if i move the annotation. But when the annotation is resized or removed, i do not get an event. If i add the stamp via the button in the toolbar ("Add or edit images"), the events are correctly sent.

If you could add a log for the event 'annotationEditorEvent' in the showcase "https://pdfviewer.net/extended-pdf-viewer/annotation-layer-api", we could verify if the events are not sent (or if they are sent, it would mean that there is a bug on my side).

**bug description - code level**
Reproducable on code level:
1. Use an editor with "annotationEditorEvent" the . For example:
```html
<ngx-extended-pdf-viewer [src]="srcBlob$ | async" (annotationEditorEvent)="onAnnotationEditorEvent($event)">
```
and in the angular component:
```ts
onAnnotationEditorEvent(event: any): void {
       console.log('Event annotationEditorEvent: ', event);
}
```

2. Add an annotation:
```ts
const annotationToDraw: PdfImageParameters = {
            urlOrDataUrl: `data:image/svg+xml;base64,${svgInBase64}`,
            page: annotation.pageIndex,
            left: x,
            bottom: y,
            right: x + annotation.width,
            top: y + annotation.height,
        };
this.pdfServiceInstance.addImageToAnnotationLayer(annotation);
```
3. Resize the annotation in the GUI => the annotationEditorEvent for 'size-changed' never occurres

**Version info**
- Version 21.3.2, tested on Chrome 127.0 on a MacOS 14.6.1

As a side note: the library is great, we already use it to let the user add stamps and also to display PDFs.

**Comment by stephanrauh** [comment]
What a strange bug! I've added the log statement locally to the showcase and, yes, I can confirm it's my bug.

**Comment by stephanrauh** [comment]
Version 21.3.5 solves your issue.

Enjoy!
Stephan

**Comment by AdrianM** [comment]
Hi Stephan, thanks a lot for the fast fix and realase! Works perfectly with 21.3.5...

Cheers
Adrian
